task/build-image-index/0.1/build-image-index.yaml

```diff
@@ -54,6 +54,9 @@ spec:
   - name: SBOM_BLOB_URL
     description: Reference of SBOM blob digest to enable digest-based verification from provenance
     type: string
+  - name: SBOM_DIGEST
+    description: A reference to the digest of the SBOM blob
+    type: string
```

Comment:
AFAIK the `SBOM_BLOB_URL` is already in `quay.io/foo/bar@sha256:digest` format. Would the UI be able to use the existing digest instead?

Reply:
The aim of this result was to make it easier for a frontend to not have to parse the existing URL and have the logic spread between backend and frontend.

Reply:
Hmm, I don't see a need to duplicate this info in the task results just for the UI. The logic should be pretty much "take the part after `@`"

Reply:
ok agree, I'll drop this and sync with the frontend team to implement this logic.